lib/hair_trigger/schema_dumper.rb

```diff
@@ -13,9 +13,11 @@ def triggers(stream)
       @connection = ActiveRecord::Base.connection
       @adapter_name = @connection.adapter_name.downcase.to_sym
 
-      db_triggers = @connection.triggers; nil
```

Comment:
i don't either, my guess it's probably an irb debugging artifact that made its way in when i wrote that commit